src/components/graph/graph.ts

```diff
@@ -138,6 +140,9 @@ export default class GraphComponent {
       .attr('width', dimensions.width)
       .attr('height', dimensions.height)
       .on('click', () => {
+        // collapses the context menu when selecting a node to move
+        d3.select('#right-click-menu').remove()
```

Comment:
create a new event type called CONTEXT_MENU_CLOSE_OR_SOMETHING and we will emit that (just like how node single clicks are being emitted to the app)